aws/resource_aws_wafv2_web_acl_logging_configuration.go

```diff
@@ -34,10 +36,66 @@ func resourceAwsWafv2WebACLLoggingConfiguration() *schema.Resource {
 				Description: "AWS Kinesis Firehose Delivery Stream ARNs",
 			},
 			"redacted_fields": {
-				Type:        schema.TypeSet,
-				Optional:    true,
-				MaxItems:    100,
-				Elem:        wafv2FieldToMatchBaseSchema(),
+				// To allow this argument and its nested fields with Empty Schemas (e.g. "method")
+				// to be correctly interpreted, this argument must be of type List,
+				// otherwise, at apply-time a field configured as an empty block
+				// (e.g. body {}) will result in a nil redacted_fields attribute
+				Type:     schema.TypeList,
+				Optional: true,
+				Computed: true,
+				MaxItems: 100,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						// TODO: remove attributes marked as Deprecated
+						// as they are not supported by the WAFv2 API
+						// in the context of WebACL Logging Configurations
+						// Reference: https://docs.aws.amazon.com/waf/latest/APIReference/API_LoggingConfiguration.html (RedactedFields)
+						"all_query_arguments": wafv2EmptySchemaDeprecated(),
```

Comment:
maybe we can skip this deprecation part (since they don't function as is?)

Reply:
Yeah, if it's never worked, I think we're OK to just remove it